### ISSUE TITLE
Unable to run auto_swipe function..

### ISSUE THREAD

**Issue by acarter881** [aj-4/tinder-swipe-bot#3]
I was unable to run the code in the auto_swipe function:

def auto_swipe(self):
        while True:
            sleep(0.5)
            try:
                self.like()
            except Exception:
                try:
                    self.close_popup()
                except Exception:
                    self.close_match()

I was able to add this line of code to the end of my code and then I was able to run everything the same way you did in the video.

bot.auto_swipe()

So the end of the code looks like this:

bot = TinderBot()
bot.login()
bot.auto_swipe()

Then, everything worked well. Okay, thanks so much Aaron. I love your videos. I just need better pictures now, and probably to pay for Tinder Plus.

**Comment by vatsalcode** [comment]
hey,I am working on this to solve this issue

**Comment by acarter881** [comment]
working on what? It has been solved. Unless you're referring to something else.

**Comment by aj-4** [comment]
yeah that's right, I ran it through the interactive shell in the video -- 
It does make more sense to have .auto_swipe() attached to the end, so everything is automatic.
best of luck with the swiping :)

**Comment by acarter881** [comment]
thanks for your comment, Aaron. I did not catch that part in the video - sorry, hehe. Thank you very much, sir!
